test/integration/apiserver/apply/apply_test.go

```diff
@@ -295,28 +296,28 @@ func TestApplyUpdateApplyConflictForced(t *testing.T) {
 		"kind": "Deployment",
 		"metadata": {
 			"name": "deployment",
-                        "labels": {"app": "nginx"}
+						"labels": {"app": "nginx"}
```

Comment:
white space here seems to have gotten worse?

Reply:
oops, fixing the whole stanza since it was pretty bad even before I made it worse. Checked the other code I added too.